src/__test__/utils/__snapshots__/postErrorToSlack.test.js.snap

```diff
@@ -0,0 +1,67 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PostErrorToSlack Posts requests correctly 1`] = `
+Array [
+  "https://slack.com/api/files.upload",
```

Comment:
make sure that none of the snapshots will inadvertently post the token / URL

Reply:
Yes, it's written out here :

https://github.com/hms-dbmi-cellenics/ui/blob/ae1cbb19ac9a4c0610026b3f417a5673151af5b2/src/__test__/utils/postErrorToSlack.test.js#L76-L81